src/Cpp/1-getting-started/1-3-7-DearImGui/Assets/Shaders/Main.ps.hlsl

```diff
@@ -0,0 +1,17 @@
+struct VSOutput
+{
+    float4 Position: SV_Position;
+    float3 Color: COLOR0;
+    float2 Uv: TEXCOORD0;
+};
+
+sampler LinearSampler : register(s0);
+
+Texture2D Texture : register(t0);
+
+float4 Main(VSOutput input): SV_Target
+{
+    float4 texel = Texture.Sample(LinearSampler, input.Uv);
+    //return float4(input.Color, 1.0f) + 0.5 * texel;
```

Comment:
This can be removed i guess

Reply:
Well, partly, this still exists in some other chapters, but those will all still require adjusting to what each chapter requires (though I'd assume each chapter after Texturing probably needs the texture sample